Modules/Microsoft365DSC/DSCResources/MSFT_AADAdministrativeUnit/MSFT_AADAdministrativeUnit.psm1

```diff
@@ -40,19 +44,12 @@ function Get-TargetResource
         [Microsoft.Management.Infrastructure.CimInstance[]]
         $ScopedRoleMembers,
 
-        <#
-        [Parameter()]
-        [Microsoft.Management.Infrastructure.CimInstance[]]
-        $Extensions,
-        #>
-
-
         #endregion
 
         [Parameter(Mandatory = $true)]
         [System.String]
         [ValidateSet('Absent', 'Present')]
-        $Ensure = 'Present',
+        $Ensure,
```

Comment:
The default value for Ensure should be Present, so that when someone doesn't specify the Ensure parameter it by default assumes present and configures the resource.

This should be configured in all functions.

Reply:
Ensure fixed